code/queue/ViewModel/src/main/java/ru/unn/agile/queue/viewmodel/ViewModel.java

```diff
@@ -35,17 +53,29 @@ public ViewModel() {
         }
     }
 
+    public List<String> getLog() {
+        return logger.getLog();
```

Comment:
Can logger be null? - If yes - It can rise NullPointerException

Reply:
How can it be? Logger can't be null, because constructor raises exception in this case.

Reply:
```
public ViewModel() {
        init();
    }
```

I suppose in this case logger will be null

Reply:
getLog function was changed